### PR TITLE
E2E: dump frr containers in pods only if running in frr mode

### PR DESCRIPTION
**Pull request by fedepaol**
When a test fails, trying to dump the frr information in native mode will result in an error. Here we check the len of containers and skip if there's only one container, assuming MetalLB is running in legacy mode.

### CODE REVIEW

File: e2etest/bgp_test.go [hunk 1]
```diff
@@ -138,6 +138,9 @@ var _ = ginkgo.Describe("BGP", func() {
 
 			speakerPods := getSpeakerPods(cs)
 			for _, pod := range speakerPods {
+				if len(pod.Spec.Containers) == 1 { // we bump only in case of frr
```
Comment (oribon) [inline e2etest/bgp_test.go:141]: nit: bump -> dump

File: e2etest/bgptests/bgp.go [hunk 2]
```diff
@@ -67,6 +67,9 @@ var _ = ginkgo.Describe("BGP", func() {
 			speakerPods, err := metallb.SpeakerPods(cs)
 			framework.ExpectNoError(err)
 			for _, pod := range speakerPods {
+				if len(pod.Spec.Containers) == 1 { // we dump only in case of frr
+					continue
```
Comment (cgoncalves) [inline e2etest/bgptests/bgp.go:71]: Mixed BGP backends are not supported. We could exit (break) the for-loop on the first speaker pod if its number of containers is equal to 1 (native mode).

**Comment by msherif1234** [comment]
dump instead of bump in PR subject too

**Pull request merged**